contracts/registry/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@textile/near-registry",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "asb",
+    "build:debug": "asb --target debug",
+    "deploy": "npm run build && npm run deploy:testnet",
+    "deploy:testnet": "near deploy bridge-registry.testnet --networkId testnet ",
```

Comment:
no init method because we want it to start empty. I could add a default provider method on init, but this seems more "open" and simple.